### PR TITLE
Fix GDPR EEACountries

### PR DESCRIPTION
**Pull request by SyntaxNode**
Resolves https://github.com/prebid/prebid-server/issues/1999.

### CODE REVIEW

File: config/config_test.go [hunk 1]
```diff
@@ -440,19 +441,12 @@ func TestFullConfig(t *testing.T) {
...
-	//Assert the NonStandardPublisherMap hash table was built correctly
-	var found bool
-	for i := 0; i < len(cfg.GDPR.NonStandardPublishers); i++ {
-		_, found = cfg.GDPR.NonStandardPublisherMap[cfg.GDPR.NonStandardPublishers[i]]
-		cmpBools(t, "cfg.GDPR.NonStandardPublisherMap", found, true)
-	}
-	_, found = cfg.GDPR.NonStandardPublisherMap["appnexus"]
-	cmpBools(t, "cfg.GDPR.NonStandardPublisherMap", found, false)
+	assert.Equal(t, []string{"pub1", "pub2"}, cfg.GDPR.NonStandardPublishers, "gdpr.non_standard_publishers")
+	assert.Equal(t, map[string]struct{}{"pub1": {}, "pub2": {}}, cfg.GDPR.NonStandardPublisherMap, "gdpr.non_standard_publishers Hash Map")
```
Comment (SyntaxNode) [inline config/config_test.go:445]: I am finally moving away from the `cmpX...` format. I really didn't want to add new helpers for arrays and maps. I miss how the description is the first argument, but that's literally all I miss.

**Review by hhhjort — APPROVED** [review]
LGTM, nice simple little fix

**Review by bsardo — APPROVED** [review]
LGTM

**Comment by SyntaxNode** [comment]
Fixed in PBS-Go 0.178.0.

**Pull request merged**